docs/index.md

```diff
@@ -132,11 +182,51 @@ $ docker-machine ip
 192.168.99.100
 ```
 
-Now you can manage as many local VMs running Docker as you please- just run
-`docker-machine create` again.
+For instance, you can try running a webserver ([nginx](https://nginx.org)) in a
+container with the following command:
+
+```
+$ docker run -d -p 8000:80 nginx
+```
+
+When the image is finished pulling, you can hit the server at port 8000 on the
+IP address given to you by `docker-machine ip`.  For instance:
+
+```
+$ curl $(docker-machine ip dev):8000
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+<style>
+    body {
+        width: 35em;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+    }
+</style>
+</head>
+<body>
+<h1>Welcome to nginx!</h1>
+<p>If you see this page, the nginx web server is successfully installed and
+working. Further configuration is required.</p>
+
+<p>For online documentation and support please refer to
+<a href="http://nginx.org/">nginx.org</a>.<br/>
+Commercial support is available at
+<a href="http://nginx.com/">nginx.com</a>.</p>
+
+<p><em>Thank you for using nginx.</em></p>
+</body>
+</html>
+```
+
+You can crete and manage as many local VMs running Docker as you please- just
```

Comment:
no fred you can crete 

![](http://mapsof.net/uploads/static-maps/Crete_integrated_map.png)